### ISSUE TITLE
what happened here ?

### ISSUE THREAD

**Issue by zarocksx** [jakes1403/gkinect#1]
Hey, does this module work ?
thx :)

**Comment by jakes1403** [comment]
Yes, it does. This repo is for Godot 3, I also have a version for Godot 4. The godot 4 module is an addon instead of a full blown engine module

**Comment by drdotpepper** [comment]
How's this going? i've just acquired a kinect v2 and i'm desperately trying to link it up with unreal/blender/godot.. anything! i'm getting close.. was wondering if you have the version for godot 4 available?

**Comment by JonnyTech** [comment]
> Yes, it does. This repo is for Godot 3, I also have a version for Godot 4. The godot 4 module is an addon instead of a full blown engine module

Ditto - would love to use a kinect with godot4 - any progress @jakes1403?

**Comment by jakes1403** [comment]
@JonnyTech @drdotpepper  Sorry I didn't reply sooner! Yeah both of these projects are pretty much dead because of the lack of free time to work on it. The Godot 4 version is here https://github.com/jakes1403/GKinect-Godot-4/tree/main
Right now, there is a simple game made with it and some other demos. You would have to tear through it on your own as there is almost no documentation, but feel free to ask me some questions if you need help.
